docs/sources/alerting/set-up/provision-alerting-resources/file-provisioning/index.md

```diff
@@ -137,7 +137,9 @@ deleteRules:
 
 Create or delete contact points in your Grafana instance(s).
 
-1. Create a YAML or JSON configuration file.
+1. Create a contact point in Grafana.
+1. Use the [Alerting provisioning API][alerting_provisioning] export endpoints to download a provisioning file for your contact point.
```

Comment:
There's no `docs/reference` section at the bottom of this page so this link won't work.